test/Generators/Microsoft.Gen.Logging/Generated/LogMethodTests.cs

```diff
@@ -231,23 +231,23 @@ public void ConstructorVariationsTests()
         Assert.Null(collector.LatestRecord.Exception);
         Assert.Equal("M1 One", collector.LatestRecord.Message);
         Assert.Equal(LogLevel.Error, collector.LatestRecord.Level);
-        Assert.Equal(1, collector.LatestRecord.Id.Id);
+        Assert.Equal(0, collector.LatestRecord.Id.Id);
```

Comment:
Just curious, why did these two change? Is that property just not being set any longer?

Reply:
Because the constructors for LoggerMessageAttribute don't contain all the overloads that LogMethodAttribute did, I had to change some of the test class definitions, which affected this.